tests/unit/test_adapter.py

```diff
@@ -947,6 +948,28 @@ def test_parse_columns_from_information_with_table_type_and_parquet_provider(sel
             },
         )
 
+    def test_describe_table_extended_2048_char_limit(self):
+        """GIVEN a list of table_names whos total character length exceeds 2048 characters
+        WHEN the environment variable DBT_DESCRIBE_TABLE_2048_CHAR_BYPASS is "true"
+        THEN the identifier list is replaced with "*"
+        """
+
+        table_names: set(str) = set([f"customers_{i}" for i in range(200)])
+
+        # By default, don't limit the number of characters
+        self.assertEqual(get_identifier_list_string(table_names), "|".join(table_names))
+
+        # If environment variable is set, then limit the number of characters
```

Comment:
Separate the different test conditions.  General rule of thumb is one assert per test (unless your setup is too expensive).

Reply:
Fixed 2fb7247f05f22c198120852647e7ee66258213ff